alembic/versions/90902c883815_create_temperature_and_city_tables.py

```diff
@@ -0,0 +1,48 @@
+"""Create temperature and city tables
+
+Revision ID: 90902c883815
+Revises: 
+Create Date: 2024-10-31 18:58:42.060376
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '90902c883815'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('city',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=False),
+    sa.Column('additional_info', sa.String(length=255), nullable=True),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('name')
+    )
+    op.create_index(op.f('ix_city_id'), 'city', ['id'], unique=False)
+    op.create_table('temperature',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=True),
```

Comment:
Consider adding a foreign key constraint to the `city_id` column in the `temperature` table to ensure referential integrity with the `city` table. This will help maintain data consistency.